### ISSUE TITLE
Reword message for not having .NET installed

### ISSUE THREAD

**Issue by puicchan** [Azure/azure-dev#1371]
In a recent user study (unmoderated getting started with azd conducted in Q4 2022), a customer tried azd with a .NET template but didn't have .NET installed. He got the error  "failed to initialize secrets at project '/path/to/project/': exec: "dotnet": executable file not found in $PATH," and did try to troubleshoot by Googling, but only looked up the first part ("azd failed to initialize secrets"), which only led him to a closed Github issue related to Azure API. He also tried searching our docs for the word "failed" but was also not able to find any information.

We believe that putting "dotnet" at the front of the error message would make it more obvious what the issue is. Propose to reword this to something like "dotnet": executable file not found in $PATH. failed to initialize secrets at project '/path/to/project/'

Presumably, this would apply to similar messages for other languages too.

**Comment by rajeshkamal5050** [comment]
Related to #821

**Comment by rajeshkamal5050** [comment]
@rajeshkamal5050 to create another issue capturing details of what validations needs to be added

**Comment by rajeshkamal5050** [comment]
> @rajeshkamal5050 to create another issue capturing details of what validations needs to be added

Added to #243 from UX perspective.

**Comment by savannahostrowski** [comment]
I'm going to close this in favour of #1449
